tests/test_normalize.py

```diff
@@ -21,53 +21,53 @@
 
 
 @pytest.mark.parametrize('normalize_trim', [normalize_trim, normalize_trim_no_shuffle])
-def test_trim(normalize_trim):
+def test_trim(normalize_fn):
     """Should trim common prefix and suffix when trim=True."""
-    assert ((25, 25), ("", "AC")) == normalize_trim(
+    assert ((25, 25), ("", "AC")) == normalize_fn(
         interval=(22, 25), alleles=(None, "AGCAC")
     )
-    assert ((24, 25), ("C", "", "CAC")) == normalize_trim(
+    assert ((24, 25), ("C", "", "CAC")) == normalize_fn(
         interval=(22, 25), alleles=(None, "AG", "AGCAC")
     )
-    assert ((23, 24), ("G", "", "GCA")) == normalize_trim(
+    assert ((23, 24), ("G", "", "GCA")) == normalize_fn(
         interval=(22, 25), alleles=(None, "AC", "AGCAC")
     )
-    assert ((22, 24), ("AG", "G", "AGCA")) == normalize_trim(
+    assert ((22, 24), ("AG", "G", "AGCA")) == normalize_fn(
         interval=(22, 25), alleles=(None, "GC", "AGCAC")
     )
 
 
 @pytest.mark.parametrize('normalize_trim', [normalize_trim, normalize_trim_no_shuffle])
```

Comment:
@gomoto You forgot to update the parametrize name to normalize_fn too. These tests won't run.